script/build.js

```diff
@@ -125,6 +125,7 @@ const ignore = require('metalsmith-ignore');
 const ignoreList = [];
 if (options.buildtype === 'production') {
   ignoreList.push('healthcare/rjsf/*');
+  ignoreList.push('pensions/application.md');
```

Comment:
Ahhh, that's why I don't see the edu forms in there. Awesome! Thanks for the clarification. 😄